docs/extend/modular-resources/examples/triton.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Add a Triton MLModel Modular Service"
+linkTitle: "Triton MLModel Modular Service"
+weight: 70
+type: "docs"
+description: "Add an ML model modular service backed by NVIDIA's Triton Server."
+tags: ["ml", "model training", "services"]
+# SMEs: Abe Winter, Andrew Morrow
+---
+
+Viam provides a MLModel service resource backed by NVIDIA's [Triton Inference Server](https://developer.nvidia.com/triton-inference-server), open-source software that makes AI model deployment and execution seamless and standardized.
+Configure this MLModel service as a [modular resource](/extend/modular-resources/) on your robot with a [`jetson` board](/components/board/jetson/) to deploy MLModels to your robot faster while consuming less compute power.
+
+Usage information is also available on [GitHub](https://github.com/viamrobotics/viam-mlmodelservice-triton).
+
+## Requirements
+
+A NVIDIA Jetson Orin board with the following installed:
+
+1. [Jetpack 5](https://developer.nvidia.com/embedded/jetpack)
+2. NVIDIA Container Runtime
+
+Run the following to install `nvidia-jetpack` and `nvidia-container` on your robot's computer:
+
+```sh { class="command-line" data-prompt="$"}
+sudo apt-get install nvidia-jetpack nvidia-container
+```
+
+Then, if you haven't done so already, create a new robot in [the Viam app](https://app.viam.com).
+[Install `viam-server` on the board](/installation/prepare/jetson-agx-orin-setup/) and connect to the robot.
+
+## Configuration
+
+{{< tabs name="Add the Triton modular service">}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot’s page in the [Viam app](https://app.viam.com).
+Click on the **Services** subtab and click the **Create service** button.
+Search for `triton`, then select the version from the [Registry](https://app.viam.com/registry).
+Give your resource a name of your choice and click **Create**.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+Add the following to your `"modules"` array:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "type": "registry",
+  "name": "viam_mlmodelservice-triton-jetpack",
+  "module_id": "viam:mlmodelservice-triton-jetpack",
+  "version": "latest"
+}
+```
+
+Add the following to your `"services"` array:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "name": "my-triton-module",
+  "type": "mlmodel",
+  "namespace": "rdk",
+  "model": "viam:mlmodelservice:triton"
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The model will now be configured with a card like the following:
+
+![The triton service card in the Viam app config builder, showing deployment options.](/extend/modular-resources/triton/triton-config-builder.png)
```

Comment:
I think the `png` should be dropped.

Reply:
Hmmm- I see how that makes sense but then how will users know what card they should be ignoring? I think it might help for confusion to have the png and more explicitly explain